app/Resources/translations/default/ModulesFaviconnotificationboAdmin.xlf

```diff
@@ -38,4 +38,33 @@ File: modules/ps_faviconnotificationbo/views/templates/admin/menu.tpl:30</note>
       </trans-unit>
     </body>
   </file>
+  <file original="modules/ps_faviconnotificationbo/ps_faviconnotificationbo.php" source-language="en-US" target-language="en" datatype="plaintext">
+    <body>
+      <trans-unit id="87cc29282288a147a6c6a8fca50ba414">
+        <source>Order Notifications on the Favicon</source>
+        <target>Order Notifications on the Favicon</target>
+        <note>Context:
+          File: modules/ps_faviconnotificationbo/ps_faviconnotificationbo.php:58</note>
+      </trans-unit>
+      <trans-unit id="421c53421b5ea9820af60bb4dba58584">
+        <source>Get notified when you get new order, new client or new message directly on your browser tab of your back office even if you are working on another page</source>
```

Comment:
I've just suggested a better wording, _Be notified of each new order, client or message directly in the browser tab of your back office, even when working on another page_, waiting for Alex D. to come back to me about it.

Reply:
New wording approved!